src/pages/settings/AppDownloadLinks.js

```diff
@@ -53,14 +81,24 @@ const AppDownloadLinksPage = (props) => {
             />
             <ScrollView style={[styles.mt5]}>
                 {_.map(menuItems, item => (
-                    <MenuItem
+                    <PressableWithSecondaryInteraction
                         key={item.translationKey}
-                        title={props.translate(item.translationKey)}
-                        icon={item.icon}
-                        iconRight={item.iconRight}
-                        onPress={() => item.action()}
-                        shouldShowRightIcon
-                    />
+                        onPressIn={() => props.isSmallScreenWidth && canUseTouchScreen() && ControlSelection.block()}
+                        onPressOut={() => ControlSelection.unblock()}
+                        onSecondaryInteraction={e => showPopover(e, item.link)}
```

Comment:
NAB, use event here?

Reply:
we have something similar here, https://github.com/Expensify/App/blob/main/src/pages/home/report/ReportActionItem.js#L154-L164

Reply:
It's named like this throughtout the code base